sdk/src/main/java/io/dapr/client/DaprClientGrpcAdapter.java

```diff
@@ -40,19 +38,30 @@
   private DaprGrpc.DaprFutureStub client;
 
   /**
-   * A utitlity class for serialize and deserialize the messages sent and retrieved by the client.
+   * A utitlity class for serialize and deserialize the transient objects.
    */
   private DaprObjectSerializer objectSerializer;
 
+  /**
+   * A utitlity class for serialize and deserialize state objects.
+   */
+  private DaprObjectSerializer stateSerializer;
+
   /**
    * Default access level constructor, in order to create an instance of this class use io.dapr.client.DaprClientBuilder
    *
-   * @param futureClient
-   * @see io.dapr.client.DaprClientBuilder
+   * @param futureClient    GRPC client
+   * @param serializer      Serializer for transient objects.
+   * @param stateSerializer Serializer for state objects.
+   * @see DaprClientBuilder
    */
-  DaprClientGrpcAdapter(DaprGrpc.DaprFutureStub futureClient, DaprObjectSerializer serializer) {
-    client = futureClient;
-    objectSerializer = serializer;
+  DaprClientGrpcAdapter(
+    DaprGrpc.DaprFutureStub futureClient,
+    DaprObjectSerializer serializer,
```

Comment:
I would rename `serializer` to `objectSerializer` here.

Reply:
It is internal class but renamed it too. Please, approve again.